userbot/utils/clients/wakatime.py

```diff
@@ -0,0 +1,61 @@
+__all__ = [
+    "WakatimeClient",
+]
+
+from base64 import b64encode
+from datetime import datetime
+from math import floor
+
+from .base import BaseClient
+
+
+def format_time(seconds: int) -> str:
+    if seconds < 60:
+        return "no coding stats for today"
+
+    hours = int(seconds // 3600)
+    minutes = int((seconds - hours * 3600) // 60)
+
+    return "{0}{1}".format(f"{hours}h" if hours > 0 else "", f" {minutes}m" if minutes > 0 else "")
```

Comment:
IIRC I have similar method somewhere in `userbot.utils`, so better not reinventing the wheel.